warehouse/query-core/src/test/java/datawave/query/jexl/JexlASTHelperTest.java

```diff
@@ -280,4 +281,250 @@ public void testNonRangeNodeNegation() throws Exception {
         Assert.assertEquals(ASTNotNode.class, nonRangeChildNodes.get(0).getClass());
     }
     
+    @Test
```

Comment:
Please change these test names to be more descriptive. Something like "testBackslashParsing" or "parseBackslashesInEqQuery"

Reply:
I will do this, but I just re-pushed the branch because some required changes were missing.

Reply:
new branch is up now.